x/nexus/exported/types.go

```diff
@@ -349,3 +349,18 @@ func FromGeneralMessage(msg GeneralMessage) WasmMessage {
 		SourceTxIndex:      msg.SourceTxIndex,
 	}
 }
+
+var _ sdk.Msg = &WasmMessage{}
+
+// ValidateBasic implements sdk.Msg
+func (m WasmMessage) ValidateBasic() error {
+	// The ValidateBasic function allows cheap stateless checks to fail msg handling early during CheckTx.
+	// This message can only be sent by the cosmwasm router as part of the amplifier integration into core,
+	// so there is no point in validating the message here.
+	return nil
+}
```

Comment:
there is no point. Nowhere in the code are there implicit assumptions about what kind of data this can hold, it's purely garbage in garbage out. The only reason this function exists is because I need to implement the sdk.Msg interface